src/main/java/executor/service/DIFactory/ObjectFactoryImpl.java

```diff
@@ -0,0 +1,81 @@
+package executor.service.DIFactory;
+
+import org.reflections.Reflections;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Type;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class ObjectFactoryImpl implements ObjectFactory {
```

Comment:
According to the contract DI:  "Factory should create all necessary services and using it for dependencies in the service facade".
![image](https://github.com/GFLCourses5/worker/assets/140727960/e066c3d2-2f19-4939-a338-b1cf52aa751b)

DI creates but does not configure classes. Please indicate how to use this.
![image](https://github.com/GFLCourses5/worker/assets/140727960/917d1338-1e44-41f2-8327-4848834b4df1)

Dee doesn't work with higher level abstractions.
![image](https://github.com/GFLCourses5/worker/assets/140727960/3027b966-b361-4e0c-9b35-f69bc275e7da)
![image](https://github.com/GFLCourses5/worker/assets/140727960/93fc8763-ad25-4c75-95d4-056a00ffbd84)

Di does not work with classes that do not have an interface, or please give specific recommendations for use.
![image](https://github.com/GFLCourses5/worker/assets/140727960/6996e763-3d9d-4a1e-a1e2-d0462ad4e957)
![image](https://github.com/GFLCourses5/worker/assets/140727960/de25b8fe-0414-4a7e-a862-7624ec4eb9cd)
![image](https://github.com/GFLCourses5/worker/assets/140727960/c547f524-d4a7-45e2-a47d-4318c8fe5b21)

If I am using DI incorrectly, please give specific recommendations for use.